src/Data/Morpheus/Server/Types/GQLType.hs

```diff
@@ -96,16 +97,21 @@ data TypeData = TypeData
 data GQLTypeOptions = GQLTypeOptions
   { fieldLabelModifier :: String -> String,
     constructorTagModifier :: String -> String,
-    -- Type used as Input will be prefixed with: Input<TypeName>
-    prefixInputType :: Bool
+    -- Construct a new type name depending on whether it is an input,
+    -- and being given the original type name
+    typeNameModifier :: Bool -> String -> String
   }
 
+defaultTypeNameModifier :: Bool -> String -> String
+defaultTypeNameModifier _ original = original
+
 defaultTypeOptions :: GQLTypeOptions
 defaultTypeOptions =
   GQLTypeOptions
     { fieldLabelModifier = id,
       constructorTagModifier = id,
-      prefixInputType = False
+      -- default is just a pass through for the original type name
+      typeNameModifier = defaultTypeNameModifier
```

Comment:
why not just `const id` ?

Reply:
nice, I'm not super familiar with `const` so I wasn't  aware that would work